arboretum/ibm_cloud_databases/README.md

```diff
@@ -0,0 +1,126 @@
+# IBM Cloud Databases library
+
+The fetchers and checks contained within this `ibm_cloud_databases` category folder are
+common tests that can be configured and executed for the purpose of generating
+compliance reports and notifications using the [auditree-framework][].  They
+validate the configuration and ensure smooth execution of an auditree instance.
```

Comment:
Remove this sentence: `They validate the configuration and ensure smooth execution of an auditree instance.`